package item;

import static org.junit.Assert.*;

import org.junit.Test;

import exception.NumberOutOfRangeException;
import test.TestUseData;

public class LotteryPlanTest {

	@Test
	public void testVisitItem() throws NumberOutOfRangeException {
		TestUseData data = new TestUseData();
		data.data1();
		LotteryPlan card = new LotteryPlan();
		assertTrue(card.visit(data.i1)==4);
	}

	@Test
	public void testVisitMeal() {
		fail("Not yet implemented");
	}

	@Test
	public void testCalculateFinalPrice() {
		fail("Not yet implemented");
	}

}
